src/getting-started/using-esp-generate.md

```diff
@@ -0,0 +1,52 @@
+# Using `esp-generate`
+
+With all the necessary tools installed, you're ready to create your first Rust project running on an Espressif chip.
+
+## Generating a Project
+
+To start, launch the interactive configuration tool by running:
+
+```shell
+esp-generate --chip esp32c3 your-project-name
+```
+
+<!-- 0.4.0 doesn't include the full interactive mode, yet --->
+Make sure to replace the chip type and project name as needed. You can also leave them out and `esp-generate` will prompt you for them.
+
+<!---
+I think a screenshot would make all this look a bit more interesting - but the drawback is it will "always" be outdated.
```

Comment:
I SC will be handy, whilst more options might come (and some might go), the general structure is likely to stay the same for a long while. Let's add a SC.